extension/app/ts/simulation/protectors/quarantine-codes.ts

```diff
@@ -32,6 +32,14 @@ export const QUARANTINE_CODES_DICT = {
 	'FAILED_CHECK': {
 		label: 'A check failed',
 		longExplanation: `A check failed to complete. Most likely because the transaction stack is too big.`
+	},
+	'SIGNATURE_CHAIN_ID_DOES_NOT_MATCH': {
+		label: 'Chain mismatch',
+		longExplanation: `The signature request is for different chain than what is the active address.`
+	},
```

Comment:
Is this actually an error?  Seems warning worthy at least, but I'm not sure it is error worthy?  One could imagine an L2 driven entirely by signatures so you never actually need to switch chains.

Reply:
yeah, one could imagine that, and maybe we should change it then. Maybe if see such use cases we could change